src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Rectangle extends Quadrangle {
+    public Rectangle(String color, int width, int height) {
+        super(color, width, height);
+    }
+
+    @Override
+    public double area() {
```

Comment:
According to the checklist, we should avoid using abstract classes to set behavior for classes. If the `area` method in `Quadrangle` is abstract and you are providing an implementation here, that's fine. However, if `Quadrangle` already provides a concrete implementation for `area` that suits `Rectangle`, then you don't need to override it here.